detox/src/android/interactions/native.js

```diff
@@ -1,20 +1,57 @@
+const _ = require('lodash');
+
 const DetoxRuntimeError = require('../../errors/DetoxRuntimeError');
 const { ScrollAmountStopAtEdgeAction } = require('../actions/native');
 const { NativeMatcher } = require('../core/NativeMatcher');
 const DetoxAssertionApi = require('../espressoapi/DetoxAssertion');
 const EspressoDetoxApi = require('../espressoapi/EspressoDetox');
 
+const SUPPORTED_MULTIVIEW_ACTIONS = new Set([
+  'getAttributes'
+]);
+
 function call(maybeAFunction) {
   return maybeAFunction instanceof Function ? maybeAFunction() : maybeAFunction;
 }
 
 class Interaction {
   constructor(invocationManager) {
     this._call = undefined;
+    this._element = undefined;
+    this._action = undefined;
     this._invocationManager = invocationManager;
   }
 
   async execute() {
+    const actionMethod = _.get(this._action, '_call.value.method', '');
```

Comment:
I don't like this code, it looks like something anti-pattern'ish.

Could you try another approach:

```js
class MultiElementActionInteraction extends Interaction {
  constructor(invocationManager, element, action) {
    super(invocationManager);
    this._call = EspressoDetoxApi.perform(call(element._call), action._call);
    this._element = element;
    this._action = action;
  }

  async execute() {
    const resultObjects = [];
    let index = 0;
    let err;
    while (!err) {
      try {
        const invocationResult = await this._invocationManager.execute(
          EspressoDetoxApi.perform(call(_.cloneDeep(this._element).atIndex(index)._call), this._action._call)
        );

        resultObjects.push(JSON.parse(invocationResult.result));

        index++;
      } catch (e) {
        err = e;
      }
    }

    if (index === 0) {
      throw err;
    }

    return resultObjects.length === 1 ?
      resultObjects[0] :
      resultObjects;
  }
}
```


and inside `NativeElement.js` do it like this?

```js
  async getAttributes() {
    const result = await new MultiElementActionInteraction(this._invocationManager, this, new actions.GetAttributes()).execute();
    return JSON.parse(result);
  }
```